src/types/JobListing.ts

```diff
@@ -0,0 +1,12 @@
+type JobListing = {
+    title:string,
```

Comment:
```suggestion
    title: string,
```

I'll add an eslint rule to make sure the spacing happens automatically :)